Module3/SQL_ASSIGNMENT_SUBMISSION.txt

```diff
@@ -0,0 +1,2321 @@
+| COMM  | int(11)      | YES  |     | NULL    |       |
++-------+--------------+------+-----+---------+-------+
+4 rows in set (0.02 sec)
+
+2. List all customers with a rating of 100. 
+mysql> SELECT CNUM,CNAME,CITY,RATING,SNUM 
+FROM CUSTOMERS 
+WHERE RATING = 100;
++------+---------+--------+--------+------+
+| CNUM | CNAME   | CITY   | RATING | SNUM |
++------+---------+--------+--------+------+
+| 2001 | Hoffman | London |    100 | 1001 |
+| 2006 | Ciemens | London |    100 | 1001 |
+| 2007 | Pereira | Rome   |    100 | 1004 |
++------+---------+--------+--------+------+
+3 rows in set (0.00 sec)
+
+3. Find the largest order taken by each salesperson on each date. 
+
+mysql> SELECT SALESPEOPLE.SNUM, SALESPEOPLE.SNAME, MAX(ORDERS.AMT) AS AMOUNT, ORDERS.ODATE  
+FROM SALESPEOPLE 
+INNER JOIN ORDERS ON ORDERS.SNUM = SALESPEOPLE.SNUM 
+GROUP BY ORDERS.ODATE, SALESPEOPLE.SNAME;
+
+
++------+---------+---------+----------+
+| SNUM | SNAME   | AMOUNT  | ODATE    |
++------+---------+---------+----------+
+| 1004 | Motika  | 1900.10 | 10/03/90 |
+| 1001 | Peel    |  767.19 | 10/03/90 |
+| 1007 | Rifkin  | 1098.16 | 10/03/90 |
+| 1002 | Serres  | 5160.45 | 10/03/90 |
+| 1003 | AxelRod | 1713.23 | 10/04/90 |
+| 1002 | Serres  |   75.75 | 10/04/90 |
+| 1001 | Peel    | 9891.88 | 10/05/90 |
+| 1002 | Serres  | 1309.95 | 10/05/90 |
++------+---------+---------+----------+
+8 rows in set (0.00 sec)
+
+4. Arrange the Order table by descending customer number. 
+
+
+mysql> SELECT ONUM,AMT,ODATE,CNUM,SNUM FROM ORDERS 
+ORDER BY CNUM DESC;
++------+---------+----------+------+------+
+| ONUM | AMT     | ODATE    | CNUM | SNUM |
++------+---------+----------+------+------+
+| 3001 |   18.69 | 10/03/90 | 2008 | 1007 |
+| 3006 | 1098.16 | 10/03/90 | 2008 | 1007 |
+| 3002 | 1900.10 | 10/03/90 | 2007 | 1004 |
+| 3011 | 9891.88 | 10/05/90 | 2006 | 1001 |
+| 3008 | 4723.00 | 10/05/90 | 2006 | 1001 |
+| 3010 | 1309.95 | 10/05/90 | 2004 | 1002 |
+| 3007 |   75.75 | 10/04/90 | 2004 | 1002 |
+| 3005 | 5160.45 | 10/03/90 | 2003 | 1002 |
+| 3009 | 1713.23 | 10/04/90 | 2002 | 1003 |
+| 3003 |  767.19 | 10/03/90 | 2001 | 1001 |
++------+---------+----------+------+------+
+10 rows in set (0.00 sec)
+
+5. Find which salespeople currently have orders in the order table. 
+mysql> SELECT DISTINCT(SALESPEOPLE.SNAME) 
+FROM SALESPEOPLE 
+INNER JOIN ORDERS
+ON SALESPEOPLE.SNUM=ORDERS.SNUM;
++---------+
+| SNAME   |
++---------+
+| Rifkin  |
+| Peel    |
+| Motika  |
+| Serres  |
+| AxelRod |
++---------+
+5 rows in set (0.00 sec)
+
+
+OR
+
+mysql> SELECT SNAME 
+FROM SALESPEOPLE 
+WHERE SNUM IN 
+(SELECT DISTINCT SNUM FROM ORDERS);
++---------+
+| SNAME   |
++---------+
+| Peel    |
+| Serres  |
+| Motika  |
+| Rifkin  |
+| AxelRod |
++---------+
+
+6. List names of all customers matched with the salespeople serving them. 
+
+mysql> SELECT CUSTOMERS.CNAME 
+FROM CUSTOMERS 
+INNER JOIN SALESPEOPLE ON CUSTOMERS.SNUM=SALESPEOPLE.SNUM;
```

Comment:
To show matching customers with salespeople you have to print the names of both salespeople and customers.

Reply:
In question no. 6 printed salespeople name and customer name.